website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md

```diff
@@ -177,9 +177,12 @@ by navigating to the URL:
 
 *Note:* If your dbt account is configured with a single-tenant cloud or on-premise installation, the `cloud.getdbt.com` domain in the URL above will be replaced with a different value.
 
-### Troubleshooting Tips
+## Troubleshooting Tips
 
-Ensure that the domain name under which user accounts exist in Azure matches the domain supplied in the SSO configuration on the dbt side.
+Ensure that the domain name under which user accounts exist in Azure matches the domain supplied in the SSO configuration on the dbt side. 
 
 <Lightbox collapsed="true" src="/img/docs/dbt-cloud/dbt-cloud-enterprise/azure/azure-get-domain.png" title="Obtaining the user domain from Azure" />
 <Lightbox collapsed="true" src="/img/docs/dbt-cloud/dbt-cloud-enterprise/azure/azure-domain-in-dbt.png" title="Adding the user domain to dbt cloud" />
+
+A way to double check after SSO has been set up would be going to this [url](https://cloud.getdbt.com/api/v2/whoami/) and assuring the `auth_provider_domain` listed is the same as the one you configured. Remember to update the domain name if you are configured with a single-tenant cloud or on-premise installation.
```

Comment:
gosh - I would really prefer it if we didn't need to direct folks to an API endpoint to find this info! I don't think the domain name is rendered anywhere inside of Cloud, but we can change that if it's going to help with verification of the SSO setup. 

@amychen1776 can you say a little bit about how often this comes up?

Reply:
@drewbanin That's valid. I know we have goals to make [this easier with this Issue](https://github.com/fishtown-analytics/dbt-cloud/issues/1563). 

Unfortunately, the API endpoint seems to be the easiest way to debug both groups and domain issues at the moment. Combined, those issues come up in most of our Azure AD implementations (I can think of 4-5 clients already)